src/lib/stepper/stepper.md

```diff
@@ -0,0 +1,128 @@
+Angular Material stepper walks users through multi-step processes by breaking them up
+into multiple logical and numbered steps. It displays progress through a sequence of steps
+and can also be used for navigation between different steps.
+
+Material stepper builds on the foundation of the CDK stepper that is responsible for the logic
+that drives a stepped workflow. Material stepper extends the CDK stepper and has Material Design
+styling.
+
+There are two stepper components: `md-horizontal-stepper` and `md-vertical-stepper`. They 
+can be used the same way. The only difference is the orientation of stepper. 
+`md-horizontal-stepper` selector can be used to create a horizontal stepper, and
+`md-vertical-stepper` can be used to create a vertical stepper. `md-step` components need to be
+placed inside either one of the two stepper components.
+
+
+### Event
+The `selectionChange` output event is emitted when the selected step changes.
```

Comment:
I'd omit the events section since the information is covered in the generated API docs (we should remove this from the tabs docs as well)